### PR TITLE
README: adding Java example to Comparison of Languages

### PR DESCRIPTION
**Pull request by andi-huber**
This change is [Reviewable](https://reviewable.io/reviews/codereport/jsource/99)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,11 @@ map ((+1) . (*2)) [0..9] -- alternative thanks to Alexandru Dinu
 // https://godbolt.org/z/5r7aEo
 iota(0, 10) | transform([](auto e) { return e * 2 + 1; })
 ```
+**Java**:
+```java
+// import java.util.stream.IntStream;
```
Comment (codereport) [inline README.md:71]:
```suggestion
```

**Review by codereport — CHANGES_REQUESTED** [review]
Looks good! Will merge after deleting the import (we omitted it for other langs)

File: README.md (hunk 1 above)
Comment (andi-huber) [reply README.md]: Yup, makes sense. Import comment removed.

**Review by codereport — APPROVED** [review]
Will merge when comment resolved :+1:

**Comment by Sebanisu** [comment]
Sorry that's my bad, I changed the same file the conflict should be easy to fix though.

**Comment by andi-huber** [comment]
No worries.

**Comment by andi-huber** [comment]
ready to merge

**Pull request merged**